internal/cache/controller_obj_cache.go

```diff
@@ -2269,6 +2269,19 @@ func (c *AviObjCache) AviObjOneVSCachePopulate(client *clients.AviClient, cloud
 						}
 					}
 				}
+				if vs["pool_group_ref"] != nil {
```

Comment:
This data is retrieved from avi controller. I think Vs doesn't have `pool_group_ref`.

Reply:
@akshayhavile child vs corresponding to httproute uses pool_group_ref in avi.